Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/modules/calculationtracker/DefaultCalculationTracker.kt

```diff
@@ -25,7 +26,8 @@ class DefaultCalculationTracker @Inject constructor(
     @AppScope private val scope: CoroutineScope,
```

Comment:
naming: I don't think this is tracking the calculation. This is tracking the Exposure Detection.

The sequence is:
1. Key Download
2. Exposure Detection (aka provideDiagnosisKeys)
3. Risk Calculation (via Exposure Summary/Exposure Windows)

Reply:
Renamed to `TrackedExposureDetections`